libtransact/Cargo.toml

```diff
@@ -137,13 +138,8 @@ contract-context-key-value = ["contract-context", "key-value-state"]
 database-lmdb = ["lmdb-zero"]
 execution = ["context", "handler", "log", "protocol-transaction", "scheduler"]
 family-command = ["handler"]
-family-command-workload = [
-    "cylinder",
-    "family-command",
-    "protocol-sabre",
-    "protocol-transaction-builder",
-    "workload",
-]
+family-command-transaction-builder = ["cylinder", "protocol-transaction-builder"]
```

Comment:
As a sub-module, this should probably pull in family-command as well?

Reply:
Added "family-command"